### PR TITLE
Edgar 22.2.2 changes

### PR DESCRIPTION
**Pull request by hermfischer**
#### Reason for change
EDGAR 22.2.2 release adds FASB 2022Q3 supplemental taxonomies and fixes a redline removal production error for some previously deployed iXBRL filings.

#### Description of change
New edgartaxonomies.xml with 2022Q3 us-gaap and srt, same added to taxonomy-compatibility.json.
New redline-removal code at EdgarRenderer/__init__.py#1495, now runs in cmd line deployment mode before instance is extracted and before R files are produced.

*Codependent with [EdgarRenderer same branch](https://github.com/Arelle/EdgarRenderer/pull/39).*

#### Steps to Test
2022Q3 taxonomies by latest SEC conf suite when published or by hand crafted samples with Q3 elements.
Redline removal requires cmd line, such as
`python3.9 arelleCmdLine.py --disclosureSystem efm-pragmatic --plugin EdgarRenderer -f {somewhere}/efm/conf/525-ix-syntax/efm/16-redlining/i12010gd/i09203gd-20081231.htm --reportXsltDissem InstanceReport.xslt`
where a Reports/dissem subdirectory has primary iXBRL doc redlines stripped as well as the zip and extracted instances.

**review**:
@Arelle/arelle

### CODE REVIEW

**Comment by aviary-wf** [comment]
## Security Insights
No security relevant content was detected by automated scans.

## Action Items
* Review PR for [security impact](https://wiki.atl.workiva.net/display/SECURITY/Development+Security+Review+Guidelines); comment "security review required" if needed or unsure
* Verify [`aviary.yaml`](https://wiki.atl.workiva.net/pages/viewpage.action?pageId=42762752#SecurityInsightFramework&Tooling-ConfiguringAviary) coverage of security relevant code
---
_Questions or Comments? Reach out on Slack: #support-infosec._

File: arelle/plugin/validate/EFM/resources/taxonomy-compatibility.json [hunk 1]
```diff
@@ -62,11 +62,12 @@
 
         "comment2": "latest us-gaap is compatible with prior year ifrs from Q1 to Q2, after Q2 must use same-year ifrs",
         "us-gaap/2021": ["@stx-2021", "srt/2021", "ifrs/2021", "cef/2021q4"],
-        "us-gaap/2022": ["@stx-2022", "srt/2022", "ifrs/2022", "cef/2022"],
+        "us-gaap/2022": ["us-gaap/2022q3", "@stx-2022", "srt/2022", "srt/2022q3", "ifrs/2022", "cef/2022"],
+        "us-gaap/2022q3": ["us-gaap/2022", "@stx-2022", "srt/2022q3", "srt/2022", "ifrs/2022", "cef/2022"],
```
Comment (austinmatherne-wk) [inline arelle/plugin/validate/EFM/resources/taxonomy-compatibility.json:66]: Is this correct?
`"us-gaap/2022"` contains `["us-gaap/2022q3",...]`
and
`"us-gaap/2022q3"` contains `["us-gaap/2022",...]`

Comment (hermfischer) [reply arelle/plugin/validate/EFM/resources/taxonomy-compatibility.json:66]: That is what was requested.

**Review by austinmatherne-wk — APPROVED** [review]

**Review by derekgengenbacher-wf — APPROVED** [review]

**Comment by derekgengenbacher-wf** [comment]
QA +10 Testing locally succeeded.

**Pull request merged**